File: 4. Primetime/primetime.c
#include <stdio.h>
#include <math.h>

int isPrime(int n){
  int i;
  for (i = 2; i <= floor(sqrt(n)); i++){
    if (n % i == 0)
      return 0;
  }
  return 1;
}

int primeTime(int l, int u){
  int count = 0;
  for (; l <= u; l++){
    if (isPrime(l)){
      count++;
    }
  }
  return count;
}

int main(){
  int l;
  int u;
  printf("Please enter a lower bound: ");
  scanf("%d", &l);
  printf("Please enter an upper bound: ");
  scanf("%d", &u);
  printf("%d", primeTime(l,u));
}

File: 4. Primetime/primetime.py
import math

def primetime(low, high):
	counter = 0
	for x in range(low, high):
		if (x == 1):
			continue
		if (isPrime(float(x))):
			counter = counter + 1
	return counter

def isPrime(n):
	for x in range(2, int(math.sqrt(n)+1)):
		if (n % x == 0):
			return False
	return True

def main():
    l = input("Please enter a lower bound: ")
	u = input("Please enter an upper bound: ")
    print(primetime(l,u))

if __name__ == "__main__":
    main()
